### ISSUE TITLE
Navbar accessibility issue.

### ISSUE THREAD

**Issue by kenlu89** [reactstrap/reactstrap#1693]
- components: `Navbar`
- reactstrap version `^7.1.0`
- import method `umd/csj/es` 
- react version `#16.8.6`
- bootstrap version `#4.3.1`

### On mobile toggle menu is not showing aria-expanded="false" when it should be.

The aria-expanded is not showing on mobile toggle after click. It should read `<div class="collapse navbar-bar" aria-expanded="false">`

This is accessibility issue.
### Steps to reproduce issue

1. import Navbar component 
2. Add few menu items
3. resize your browser into mobile version.
4. Inspect the menu toggle button and try to click no aria-expanded appear under `<div class="collapse">` tag

### Here is my code. Just using the example code from Navbar component 

```
`import React, { useState } from 'react';
import { Collapse, Navbar, NavbarToggler, NavbarBrand, Nav, NavItem, NavLink } from 'reactstrap';

const Example = (props) => {
  const [collapsed, setCollapsed] = useState(true);

  const toggleNavbar = () => setCollapsed(!collapsed);

  return (
    <div>
      <Navbar color="faded" light>
        <NavbarBrand href="/" className="mr-auto">reactstrap</NavbarBrand>
        <NavbarToggler onClick={toggleNavbar} className="mr-2" />
        <Collapse isOpen={!collapsed} navbar>
          <Nav navbar>
            <NavItem>
              <NavLink href="/components/">Components</NavLink>
            </NavItem>
            <NavItem>
              <NavLink href="https://github.com/reactstrap/reactstrap">GitHub</NavLink>
            </NavItem>
          </Nav>
        </Collapse>
      </Navbar>
    </div>
  );
}
export default Example;
```

**Comment by TheSharpieOne** [comment]
This was fixed in #1657. Please update your version of reactstrap.

**Comment by kenlu89** [comment]
@TheSharpieOne  Thank you. What command use in react to update reactstrap to latest version?

**Comment by TheSharpieOne** [comment]
`npm install reactstrap@latest --save`
